babel.config.ts

```diff
@@ -20,6 +20,17 @@ const config: TransformOptions = {
     ],
     '@babel/preset-typescript',
   ],
+  overrides: [{
```

Comment:
This override is necessary because I ran into a runtime error from babel:
`importSource cannot be set when runtime is classic`

We are using React 17's new JSX transform with automatic runtime (see line 10 of this same file). Further, we have set importSource to be `@emotion/react` (see line 11). The problem is that the importSource option is only available to automatic runtime, and for some reason our Storybook compiling falls back to classic runtime (perhaps due to lack of support for React 17 from the version of Storybook that we're using). So, the importSource command cannot be fulfilled, leading to the error.

The solution: set an overriding Babel config object only applies to Storybook, telling Babel to use automatic runtime but with no specific, developer-defined importSource. This resolves the issue above.